cfme-performance/utils/workloads.py

```diff
@@ -7,3 +7,31 @@ def get_capacity_and_utilization_scenarios():
             len(cfme_performance['workloads']['test_cap_and_util']['scenarios']) > 0):
         return cfme_performance['workloads']['test_cap_and_util']['scenarios']
     return []
+
+
+def get_refresh_rapid_scenarios():
+    if (cfme_performance['workloads']['test_refresh_rapid']['scenarios'] and
+            len(cfme_performance['workloads']['test_refresh_rapid']['scenarios']) > 0):
+        return cfme_performance['workloads']['test_refresh_rapid']['scenarios']
+    return []
+
+
+def get_refresh_wait_scenarios():
+    if (cfme_performance['workloads']['test_refresh_wait']['scenarios'] and
+            len(cfme_performance['workloads']['test_refresh_wait']['scenarios']) > 0):
+        return cfme_performance['workloads']['test_refresh_wait']['scenarios']
+    return []
+
+
+def get_refresh_vm_rapid_scenarios():
+    if (cfme_performance['workloads']['test_refresh_vm_rapid']['scenarios'] and
+            len(cfme_performance['workloads']['test_refresh_vm_rapid']['scenarios']) > 0):
+        return cfme_performance['workloads']['test_refresh_vm_rapid']['scenarios']
+    return []
+
+
+def get_refresh_vm_wait_scenarios():
+    if (cfme_performance['workloads']['test_refresh_vm_wait']['scenarios'] and
+            len(cfme_performance['workloads']['test_refresh_vm_wait']['scenarios']) > 0):
+        return cfme_performance['workloads']['test_refresh_vm_wait']['scenarios']
+    return []
```

Comment:
Please add a few example scenarios to the workloads section of cfme_performance.yml for each scenario (Similiar to what we have for Cap&Util scenario.